common/src/main/java/com/wynntils/features/user/overlays/MiniMapOverlayFeature.java

```diff
@@ -106,17 +110,19 @@ public void render(PoseStack poseStack, float partialTicks, Window window) {
 
             // disable mask
             switch (maskType) {
-                case Rectangular -> {
-                    RenderSystem.disableScissor();
-                }
+                case Rectangular -> RenderSystem.disableScissor();
                     // case Circle -> {
                     // TODO
                     // }
             }
 
             // TODO cursor
 
-            // TODO Border rendering
+            // render border
+            switch (maskType) {
```

Comment:
Isn't it a bit weird to have 3 switches of the same thing?

Reply:
I don't see an alternative as of now, but I can replace it with if statements if desired

Reply:
I am talking about only having 1 switch and doing everything needed for rendering setup there.

Reply:
I think we need at least 2 switches as they need to enable and disable the mask, but the last two maybe can be merged a bit clunkily.

Reply:
Ah. Okay.